test/fixtures/fails.yml

```
@@ -0,0 +1,3 @@
+fail_default:
+  user:
+  data: 'data'
```

```suggestion
  data: '{}'
```
Conviene que sea JSON válido, por cualquier cosa.

---

bien